openshift/pgslice/docker/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM ankane/pgslice:v0.6.1
+FROM artifacts.developer.gov.bc.ca/docker-remote/ubuntu:24.04
```

Comment:
Good idea to move away from `ankane/pgslice`.

Reply:
Yeah although this image still installs the package. What I like about the package though is it's very small and would be relatively easy to replicate necessary functionality if it ever goes unsupported for a long time.